go/pkg/pwapi/api_challenge-subscription-validate.go

```diff
@@ -180,6 +180,15 @@ func (svc *service) ChallengeSubscriptionValidate(ctx context.Context, in *Chall
 			return err
 		}
 
+		// update team score
+		err = tx.Model(&pwdb.Team{}).
+			Where("id = ?", subscription.TeamID).
```

Comment:
```suggestion
			Where(pwdb.Team{ID: subscription.TeamID}).
```

a little bit better (go type instead of string).